### PR TITLE
fixing the width argument description of the function draw_networkx

### PR DESCRIPTION
**Pull request by amamory**
It's a simple correction in the docstring description in the draw_networkx.

### CODE REVIEW

**Review by dschult — APPROVED** [review]
This change makes the description match that used in the `draw_networkx_edges` function which it should since the value is just passed right through to that function.

Thank you!

File: networkx/drawing/nx_pylab.py [hunk 1]
```diff
@@ -192,8 +192,8 @@ def draw_networkx(G, pos=None, arrows=True, with_labels=True, **kwds):
     linewidths : [None | scalar | sequence]
        Line width of symbol border (default =1.0)
 
-    width : float, optional (default=1.0)
-       Line width of edges
+    width : float, or array of floats
+       Line width of edges (default=1.0)
```
Comment (rossbar) [inline networkx/drawing/nx_pylab.py:196]:
```suggestion
    width : float or array of floats (default=1.0)
       Line width of edges
```
I suggest leaving the default on the parameter-type line to stay consistent with all of the other entries.

**Review by rossbar — COMMENTED** [review]
One minor change but other than that LGTM, thanks @amamory

File: networkx/drawing/nx_pylab.py (hunk 1 above)
Comment (dschult) [reply networkx/drawing/nx_pylab.py:196]: Yes, the defaults should be on the first line -- but I see that the `linewidth` parameter just above this one has it on the second line and the `width` parameter to `draw_networkx_edges()` is exactly the same as this one is now -- default on the second line. Looks like we should clean up the docstrings in this module.

**Comment by dschult** [comment]
OK... this is ready for review -- much more change than the original... Thanks @amamory for the nudge! :}

File: networkx/drawing/nx_pylab.py [hunk 2]
```diff
@@ -5,15 +5,17 @@
...
 
-matplotlib:     http://matplotlib.org/
-
-pygraphviz:     http://pygraphviz.github.io/
-
+See Also
+--------
+ - matplotlib: http://matplotlib.org/
+ - matplotlib's scatter: https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.pyplot.scatter.html
+ - matplotlib's FancyArrowPatch: https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.patches.FancyArrowPatch.html
```
Comment (rossbar) [inline networkx/drawing/nx_pylab.py:17]:
```suggestion
 - :doc:`matplotlib <matplotlib:index>`
 - :func:`matplotlib.pyplot.scatter`
 - :obj:`matplotlib.patches.FancyArrowPatch`
```
We could use intersphinx for these links to make sure they remain up-to-date.

**Review by rossbar — COMMENTED** [review]
One final touchup to use intersphinx for a couple of the matplotlib links. Thanks @dschult & @amamory !

**Comment by rossbar** [comment]
This will also need a rebase on master. I'd be happy to do so, but unfortunately I don't think I can since the PR is originating from @amamory 's `master` branch.

**Comment by dschult** [comment]
I was able to push to that branch.

Very nice intro for me to intersphinx!!   
Now I know enough to want to go look up more.
Thanks!

**Comment by amamory** [comment]
hi guys, the pull was closed, so i believe i can delete my fork. could you confirm ?

**Comment by dschult** [comment]
Yup!   That is fine!  
Thank you!!

**Pull request merged**